cmd/queue/main.go

```diff
@@ -353,9 +392,16 @@ func main() {
 		StatChan:   statChan,
 	}, time.Now())
 
+	coreProbe, err := parseProbe(*ucProbe)
+	if err != nil {
+		logger.Fatalw("Queue container failed to parse readiness probe", zap.Error(err))
+	}
+
+	rp := readiness.NewProbe(coreProbe, logger.With(zap.String(logkey.Key, "readinessProbe")))
```

Comment:
Consider exposing `readiness.EncodeProbe()` and `readiness.DecodeProbe` to [de]serializing the probe.